source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -569,56 +562,71 @@ namespace RendererQml
 				type,
 				choice->GetTitle(),
 				choice->GetValue(),
-				fontSize,
 				isWrap,
 				isVisible,
-				isChecked,
-                textColor));
+				isChecked));
 		}
 
 		RendererQml::ChoiceSet choiceSet(id,
 			input->GetIsMultiSelect(),
 			input->GetChoiceSetStyle(),
 			parsedValues,
 			choices,
-			input->GetPlaceholder(),
-            backgroundColor);
+			input->GetPlaceholder());
 
 		if (CheckBoxType::ComboBox == type)
 		{
-			return GetComboBox(choiceSet);
+			return GetComboBox(choiceSet,context);
 		}
 		else
 		{
-			return GetButtonGroup(choiceSet);
+			return GetButtonGroup(choiceSet, context);
 		}
 
 		std::shared_ptr<QmlTag> uiColumn;
 		return uiColumn;
 	}
 
-	std::shared_ptr<QmlTag> AdaptiveCardQmlRenderer::GetComboBox(ChoiceSet choiceset)
+	std::shared_ptr<QmlTag> AdaptiveCardQmlRenderer::GetComboBox(ChoiceSet choiceset, std::shared_ptr<AdaptiveRenderContext> context)
 	{
 		auto uiComboBox = std::make_shared<QmlTag>("ComboBox");
-				
+		const auto textColor = context->GetColor(AdaptiveCards::ForegroundColor::Default, false, false);
+		const auto backgroundColor = context->GetRGBColor(context->GetConfig()->GetContainerStyles().defaultPalette.backgroundColor);
+
 		uiComboBox->Property("id",choiceset.id);
 		uiComboBox->Property("textRole", "'text'");
 		uiComboBox->Property("valueRole", "'value'");
 		uiComboBox->Property("width", "parent.width");
 		//TODO : Add Height
-				
-		uiComboBox->Property("model", GetModel(choiceset.choices));
 
-        const auto textColor = choiceset.choices[0].textColor;
+		auto dropIcon = std::make_shared<QmlTag>("Image");
+		dropIcon->Property("source", Formatter() << "\"" << RendererQml::arrow_down_12 << "\"");
+		dropIcon->Property("anchors.right", "parent.right");
```

Comment:
is this needed?

Reply:
yes to keep the icon in the right most corner